config.toml

```diff
@@ -0,0 +1,56 @@
+[taker_config]
+# relatively low value for now so that its easier to test without having to wait too much
+# right now only the very brave will try coinswap out on mainnet with non-trivial amounts
```

Comment:
I think this file should be removed now?

Reply:
The only reason why I kept it, was to check case, when some fields are not available.